src/plugins/minimal_scene/EngineToQtInterface.cc

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include "EngineToQtInterface.hh"
+
+#include <gz/common/Console.hh>
+#include <gz/rendering/Camera.hh>
+#include <gz/rendering/RenderEngine.hh>
+
+#include <QOpenGLExtraFunctions>
+
+#include <GL/glcorearb.h>
```

Comment:
hmm [windows build](https://build.osrfoundation.org/job/ign_gui-pr-win/1744/) complains about this missing include file
```
fatal error C1083: Cannot open include file: 'GL/glcorearb.h': No such file or directory
```

Reply:
We're lucky, `GL/glcorearb.h` is not needed anymore. Line removed.